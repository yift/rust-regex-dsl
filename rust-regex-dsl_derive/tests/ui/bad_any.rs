pub use rust_regex_dsl::regex_dsl;

fn main() {
    let regex = regex_dsl! {
        any,
    };
    let regex = regex_dsl! {
        any(),
    };
    let regex = regex_dsl! {
        any("one"),
    };
    let regex = regex_dsl! {
        any(regex("["), "two"),
    };
    println!("{}", regex);
}