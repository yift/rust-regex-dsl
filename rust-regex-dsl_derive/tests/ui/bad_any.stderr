error: Unknown word: any
 --> tests/ui/bad_any.rs:5:9
  |
5 |         any,
  |         ^^^

error: unexpected end of input, Concat must have at least two elements
 --> tests/ui/bad_any.rs:8:13
  |
8 |         any(),
  |             ^

error: unexpected end of input, Concat must have at least two elements
  --> tests/ui/bad_any.rs:11:18
   |
11 |         any("one"),
   |                  ^

error: regex parse error:
           [
           ^
       error: unclosed character class
  --> tests/ui/bad_any.rs:14:19
   |
14 |         any(regex("["), "two"),
   |                   ^^^
